packages/fixie/src/client.ts

```diff
@@ -0,0 +1,156 @@
+/**
+ * A client to the Fixie AI platform.
+ */
+export class FixieClient {
+  url!: string;
+  apiKey!: string;
+
+  /** Use Create() instead. */
+  constructor() {}
+
+  /**
+   * Create a new Octopus Farmer client.
+   * @param url The URL of the Fixie API server.
+   */
+  public static async Create(url: string): Promise<FixieClient> {
+    const client = new FixieClient();
+    client.url = url;
+    const apiKey = process.env.FIXIE_API_KEY;
+    if (!apiKey) {
+      throw new Error(
+        'Must set the FIXIE_API_KEY environment variable. This can be found at: https://app.fixie.ai/profile'
+      );
+    }
+    client.apiKey = apiKey;
+    return client;
+  }
+
+  async request(path: string, body?: any): Promise<any> {
+    let res;
+    if (body) {
+      body = JSON.stringify(body);
+      res = await fetch(`${this.url}${path}`, {
+        method: 'POST',
+        headers: {
+          Authorization: `Bearer ${this.apiKey}`,
+          'Content-Type': 'application/json',
+        },
+        body,
+      });
+    } else {
+      res = await fetch(`${this.url}${path}`, {
+        method: 'GET',
+        headers: {
+          Authorization: `Bearer ${this.apiKey}`,
+        },
+      });
+    }
+    if (!res.ok) {
+      throw new Error(`Failed to access Fixie API ${this.url}${path}: ${res.statusText}`);
+    }
+    return res.json();
+  }
+
+  // Read the state of the current game.
+  async userInfo(): Promise<any> {
+    return this.request('/api/user');
+  }
+
+  async listCorpora(): Promise<any> {
+    return this.request('/api/v1/corpora');
+  }
+
+  async getCorpus(corpusId: string): Promise<any> {
+    return this.request(`/api/v1/corpora/${corpusId}`);
+  }
+
+  async createCorpus(name?: string): Promise<any> {
```

Comment:
Instead of `Promise<any>`, it would be ideal if we could type out the API's return interface.

If that's too onerous, we should use `unknown` or `Jsonifiable` instead of `any`.

Reply:
Eventually Mike and I will have some kind of OpenAPI spec generated from the Fixie REST APIs that could be used for this. I'll use Jsonifiable for now.